facial-age-estimator/src/facial-age-estimator-model.js

```diff
@@ -0,0 +1,98 @@
+/* globals tf */
+import * as tf from '@tensorflow/tfjs-node'
+
+let modelPath = null
+
+if (!process.rollupBrowser) {
+  modelPath = `file://${__dirname}/../model/model.json`
+} else {
+  modelPath = 'https://max-assets.s3.us.cloud-object-storage.appdomain.cloud/facial-age-estimator/tf_js/0.1/model.json'
```

Comment:
i am unable to access this? is that the correct URL?
and i don't think this line will ever be reached since `process.rollBrowser` is not configured anywhere

Reply:
Sorry, I haven't uploaded the file yet.

Reply:
not a problem. i just wasnt sure how far you had gotten along